README.md

```diff
@@ -34,6 +34,32 @@ Run the application by executing:
 This repo is now integrated with the rse-cft-library. For more information see the bootWithCCD task in build.gradle.
 Extra compose files can be provided in src/cftlib/resources/docker.
 
+#### Cftlib - Docker Desktop v4.30
+
+**_First Time Setup_**
+
+As part of this setup the definitions repository has been added as a git submodule.
+1. `cd definitions`
+2. `git submodule init` & `git submodule update`
+4. `git checkout master` & `git pull`

```

Comment:
can we not do this when we run bootWithCcd so we don't have to do this manually ?